hathor/consensus/consensus.py

```diff
@@ -75,11 +76,13 @@ def update(self, base: BaseTransaction) -> None:
         assert meta.validation.is_valid()
         try:
             self._unsafe_update(base)
-        except Exception:
-            meta.add_voided_by(self._settings.CONSENSUS_FAIL_ID)
```

Comment:
I feel we should keep adding this marker. It's important for debugging.

Reply:
Sure, I brought back that transaction marking

Reply:
Thanks!